$MirOS: ports/print/cups/patches/patch-filter_imagetoraster_c,v 1.2 2006/06/21 19:25:35 bsiegert Exp $
--- filter/imagetoraster.c.orig	Mon May 15 05:01:52 2006
+++ filter/imagetoraster.c	Tue Jul  4 17:55:16 2006
@@ -258,7 +258,7 @@ main(int  argc,				/* I - Number of comm
       close(mypipes[1]);
 
       execlp("pstoraster", argv[0], argv[1], argv[2], argv[3], argv[4], argv[5],
-             NULL);
+             (char *)NULL);
       perror("ERROR: Unable to exec pstoraster");
       return (errno);
     }
@@ -287,7 +287,7 @@ main(int  argc,				/* I - Number of comm
     */
 
     execlp("imagetops", argv[0], argv[1], argv[2], argv[3], argv[4], argv[5],
-           argv[6], NULL);
+           argv[6], (char *)NULL);
     perror("ERROR: Unable to exec imagetops");
     return (errno);
   }
